src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -695,6 +695,17 @@ public Optional<List<IdentifiableAttributes>> exportFilter(UUID id, UUID network
         return getFilter(id).map(filter -> getIdentifiableAttributes(filter, networkUuid, variantId));
     }
 
+    public Map<String, List<Integer>> containersListCount(Map<String, List<UUID>> filtersIdsMap, UUID networkUuid, String variantId) {
+        Objects.requireNonNull(filtersIdsMap);
+        Map<String, List<Integer>> resultMap = new HashMap<>();
+        for (String key : filtersIdsMap.keySet()) {
+            List<UUID> uuids = filtersIdsMap.get(key);
+            List<FilterEquipments> filtersEquipments = exportFilters(uuids, networkUuid, variantId);
```

Comment:
Do not use `exportFilters`

    public Map<String, List<Integer>> getIdentifiablesCount(Map<String, List<UUID>> ids, UUID networkUuid, String variantId) {
        Objects.requireNonNull(ids);
        return ids.entrySet().stream()
                .map(entry -> Map.entry(entry.getKey(), getFilters(entry.getValue()).stream().map(f -> getIdentifiableAttributes(f, networkUuid, variantId).size()).toList()))
                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
    }

Reply:
done